src/input.rs

```diff
@@ -238,25 +236,9 @@ impl Action {
             },
             Action::Command(ref program, ref args) => {
                 trace!("running command: {} {:?}", program, args);
-
-                #[cfg(not(windows))]
-                let spawned = Command::new(program)
-                    .args(args)
-                    .before_exec(|| {
-                        // Detach forked process from Alacritty. This will cause
-                        // init or whatever to clean up child processes for us.
-                        unsafe { ::libc::daemon(1, 0); }
-                        Ok(())
-                    })
-                    .spawn();
-
-                #[cfg(windows)]
-                let spawned = Command::new(program)
-                    .args(args)
-                    .spawn();
-
-                match spawned
-                {
+                //.collect::<Vec<_>>();
+                let spawned= start_daemon(program,args);
+                match spawned{
```

Comment:
This can be rewritten to just the following:

```rust
match start_daemon(program, args) {
```

Please make sure that spaces are set appropriately to ensure readability of the code.